dotcom-rendering/src/palette.ts

```diff
@@ -862,9 +998,17 @@ const paletteColours = {
 		light: standfirstLinkBorderLight,
 		dark: standfirstLinkBorderDark,
 	},
-	'--follow': {
-		light: followLight,
-		dark: followDark,
+	'--follow-icon-fill': {
+		light: followIconFillLight,
+		dark: followIconFillDark,
+	},
+	'--follow-icon-background': {
+		light: followIconBackgroundLight,
+		dark: followIconBackgroundDark,
+	},
```

Comment:
I don’t fully understand the distinction between these two colours. Could you explain?

Reply:
had no idea this was a thing. great shout

Reply:
It required quite intimate knowledge of [how SVG `fill-rule` works](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-rule). I would love if Source offered cut-out versions of icons, but it’s definitely not a blocker for this PR!

Reply:
Going to leave as is for now until a cutout option is made available on the Source components which is the optimum solution